.travis.yml

```diff
@@ -0,0 +1,77 @@
+sudo: required
```

Comment:
Do you actually need sudo?  We need it in 2.0 for installing mingw (because the version in the standard repo is too old), but you don't seem to need it (yet). If you don't need it, don't specify it. Your builds will then be faster (there is only a limited number of 'sudo' slaves available).

Reply:
Oh ok. Thank for the hint. I wasn't sure about it either, but I did not investigate further. I'll remove it.